keg_auth/templates/keg_auth/crud-addedit.html

```diff
@@ -4,3 +4,23 @@
 {% block form %}
 {{ horizontal.form(form, action_text=action_button_text, cancel_url=cancel_url) }}
 {% endblock %}
+
+{% block scripts %}

```

Comment:
This adds permission specific JS to every add/edit form that we use. Not the end of the world but not ideal. It might be better to move this and the `MultiCheckboxField` also over to keg elements, and then add a `data-target` attribute to the select all checkbox so it is generic instead of being tied directly to `permission_ids`.